src/yaml.rs

```diff
@@ -126,13 +126,7 @@ mod test {
 		let mut variables = HashMap::new();
 		variables.insert("bar", "aap");
 		variables.insert("baz", "noot");
-		let_assert!(Ok(parsed) = from_str(
-			concat!(
-				"bar: $bar\n",
-				"baz: $baz/with/stuff\n",
-			),
-			&variables,
-		));
+		let_assert!(Ok(parsed) = from_str(concat!("bar: $bar\n", "baz: $baz/with/stuff\n",), &variables,));
```

Comment:
And this :)

Reply:
Don't forget this type of test. The resulting `concat!` is rather strange ^^

We *could* just remove the concat here, but I do prefer the readability of having the line breaks in the source code.